### PR TITLE
change headres, add required files

### PR DESCRIPTION
**Pull request by MajdT51**

### CODE REVIEW

File: NOTICE.md [hunk 1]
```diff
@@ -1,8 +1,14 @@
-# Notices
+# Notices for Eclipse Tractus-X
 
-This content is part of [CatenaX](https://catena-x.net). 
+This content is produced and maintained by the Eclipse Tractus-X project.
 
-* Project home: https://github.com/catenax-ng
+* Project home: https://projects.eclipse.org/projects/automotive.tractusx
+
+See the AUTHORS file(s) distributed with this work for additional information regarding authorship.
```
Comment (jpbu) [inline NOTICE.md:7]: I believe we did not include the AUTHORS file, so we should add one or remove this sentence.

File: SECURITY.md [hunk 2]
```diff
@@ -1,11 +1,12 @@
...
+## Reporting a Vulnerability
 
+Please report a found vulnerability here:
+[https://www.eclipse.org/security/](https://www.eclipse.org/security/)
 
 ## Reporting a bug in Catena-X
 
-
-
 Report security bugs in Catena-X to "dl_CoP_IT_Security@catena-x.net".
```
Comment (jpbu) [inline SECURITY.md:10]: Will this be kept in parallel to the Eclipse security reporting? The distinction between "vulnerability" and "security bug" might be confusing.

**Review by jpbu — COMMENTED** [review]
I just had a brief look at the changes and believe @matthiashub-porsche can better judge this, two comments related to things I stumbled upon. I also found a few references to the catena-x GitHub repository link in the README, OpenAPI, Routing.

File: SECURITY.md (hunk 2 above)
Comment (matthiashub-porsche) [reply SECURITY.md:10]: Yes, we were asked to put both in

**Comment by jpbu** [comment]
Thanks for the updates - I believe this branch must now be updated to the latest develop version (there were some new classes added in other merges and there now is a conflict)

**Comment by matthiashub-porsche** [comment]
> Thanks for the updates - I believe this branch must now be updated to the latest develop version (there were some new classes added in other merges and there now is a conflict)

My suggestion is to wait with the merge when we are in the Eclipse repo, the request is already out and it was mentioned that within a week that could be cleared.

**Pull request merged**